XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,75 @@
+<Page
+    x:Class="AppUIBasics.ControlPages.CornerRadiusPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:Windows10version1809="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract, 7)"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:AppUIBasics"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+    <StackPanel>
+        <TextBlock IsTextSelectionEnabled="True">
+            <LineBreak />
+            <Run Text="For basic UI elements like buttons that users see on screen at all times," />
+            <Run Text="the default radius value is" />
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for is" />
+            <Run FontWeight="Bold" Text="4px" />
+            <Run Text="." />
+        </TextBlock>
+        <TextBlock>
+            <Run Text="These demos require Windows 10 Version 1809 or later to work properly. To set a conditional namespace, check out our" />
```

Comment:
This text should be handled by setting the MinimumUniversalAPIContract; I don't think we'll need this extra text.